src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar.domain
+
+class Cars(
+    private val cars: List<Car>
+) {
+    fun getCars() = cars
+
+    fun race() = cars.map { it.moveOrNot() }
```

Comment:
리턴값이 불필요하니 map 대신에 forEach를 사용해도 되겠네요! 저는 이런 상황에서 함수 시그니처 타입을 꼭 명시해두는걸 선호해요.
```suggestion
    fun race(): Unit = cars.map { it.moveOrNot() } // 컴파일 에러가 나기에 의도치 않은 타입을 리턴하는 것을 막을 수 있음
```